recipes/gatiab/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "gatiab" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/gatiab-{{ version }}.tar.gz
+  sha256: 0fdd77decd846cff54f51542a1e370385ef8fe0936644a6c71ae95f2aed570e4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9.19
```

Comment:
Please use the `python_min` convention for `noarch: python` packages shown in:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python